about.html

```diff
@@ -72,13 +72,26 @@ <h2>2023-19-5(Fri) <br><span class="span">CEO, Jalloh ~</span> StartUp TidCoin</
       <div class="image">
         <img class="mobile" src="images/image-mockups.png" alt="image-mockups" width="100%">
       </div>
+
```

Comment:
- [ ] Great job working on the third section but I think you should add the heading like the given design since i can't see in your design so please add to match the given design, and regarding to the images I think it would be great if you decrease the width for it to match the given design so please fix it as well 💯🙏🏻


- PLease notice the left and right margin in the given design and please fix this as well🙏🏻💯
- Your
![image](https://github.com/aradradev/easyBankTC/assets/58553711/220d51d7-84be-476c-9d57-a3e385255e21)


- Given
![image](https://github.com/aradradev/easyBankTC/assets/58553711/0a07c535-c471-4438-aa98-5542e1cba458)